### DISCUSSION TITLE
[chatcommands] Add shorthands for Sol Heredit/Fortis Colosseum

### DISCUSSION THREAD

**Discussion by Nuunnyyaahh** [runelite/runelite#17951] [Ideas]
When completing Fortis Colosseum on a PC or device that does not have or is not using Runelite, the KC/PB does not show on Runelite as we know. Using the PoH adventure log counters page will usually fix this issue by updating Runelite to show your current KC/PB. For Fortis Colosseum/Sol Heredit, it will update your KC on Sol Heredit and PB for Fortis Colosseum, but not your PB for Sol Heredit. If you were to complete the Fortis Colosseum while using Runelite, it shows your KC and PB on Sol Heredit as well as your PB for Fortis Colosseum. There needs to be mapping for Sol Heredit > Fortis Colosseum for the PoH adventure logs counters page to update Runelite properly to show your PB on Sol Heredit when getting the kill while not using Runelite.

I attached some screenshots of the PoH adventure log and PB commands in Runelite to show the difference. The Fortis Colosseum was completed on 2 different accounts. One via Runelite and one without Runelite. Both were updated on Runelite via the PoH adventure logs counters page.

![2](https://github.com/runelite/runelite/assets/61466272/4d0c0bde-8f85-444f-a689-80bdfee81449)
![22](https://github.com/runelite/runelite/assets/61466272/e042b0ba-3ae3-41dc-9b4f-4cb27b5cfea4)
![1](https://github.com/runelite/runelite/assets/61466272/6987babd-63d4-491c-bb27-039627abc119)
![11](https://github.com/runelite/runelite/assets/61466272/05737120-ccf5-4235-b4fb-ad86cd30ff84)

**Comment by Alexsuperfly** [comment]
added in #17828
